packages/waku/src/lib/middleware/rsc/worker-impl.ts

```diff
@@ -213,17 +214,8 @@ const resolveClientEntry = (
     filePath = filePath.slice('file://'.length);
   }
   filePath = resolveClientPath?.(filePath) || filePath;
-  let root = path.join(
-    config.rootDir,
-    command === 'dev' ? config.srcDir : config.distDir,
-  );
-  if (path.sep !== '/') {
-    // HACK to support windows filesystem
-    root = root.replaceAll(path.sep, '/');
-    if (filePath[0] === '/') {
-      filePath = filePath.slice(1);
-    }
-  }
+  let root = path.join(config.rootDir, command === 'dev' ? '' : config.distDir);
```

Comment:
Hmm, root is not `srcDir` in DEV? I would like to see `srcDir` and `distDir` symmetrically.

Reply:
I'm not sure how to achieve this to be honest. Do you have any idea? As we talked the purpose was to make something like reading files working, and It's working now (at least in the entries.ts). 

If you have any other reasons for that, it'd help understand the motivation and hopefully find a solution. If you have a solution also let me know.

Reply:
As we talked in discord, can we skip this and face it when we have a problem? Let me know.

Reply:
Yes, please leave some comments:
```suggestion
  // FIXME We want to make `root` symmetric for 'srcDir' and 'distDir', but we don't know how to achieve it.
  let root = path.join(config.rootDir, command === 'dev' ? '' : config.distDir);
```